.pre-commit-config.yaml

```diff
@@ -0,0 +1,42 @@
+exclude: ^archive/.*|.*/test-data/.*|.*/vendor/.*|.*\.svg$
+minimum_pre_commit_version: "2.15.0"
+repos:
+  - repo: "https://github.com/pre-commit/pre-commit-hooks"
+    rev: "v4.0.1"
+    hooks:
+      - id: "check-case-conflict"
+      - id: "mixed-line-ending"
+      - id: "end-of-file-fixer"
+      - id: "trailing-whitespace"
+      - id: "check-json"
+      - id: "check-yaml"
+      - id: "check-added-large-files"
+      - id: "detect-aws-credentials"
+        args: [ "--allow-missing-credentials" ]
+      - id: "detect-private-key"
```

Comment:
Do you really need all these checks? For instance, we do not use aws. Why do we detect aws credentials?

Reply:
ok, remove aws